### PR TITLE
Rabbit connection

### PR DESCRIPTION
**Pull request by mkpurcell**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+numpy==1.17.4
+h5py==2.10.0
+tensorflow==1.15
```
Comment (chalianwar) [inline requirements.txt:3]: Is this the version we want for default?

Comment (chalianwar) [reply requirements.txt:3]: @Yi-Zoey and @shashank215r

Comment (mkpurcell) [reply requirements.txt:3]: @chalianwar do we have answer for this yet?

**Review by chalianwar — APPROVED** [review]
lGTM! thanks!

**Pull request merged**